cumulus/parachains/runtimes/people/people-westend/src/xcm_config.rs

```diff
@@ -91,8 +82,8 @@ pub type LocationToAccountId = (
 	SiblingParachainConvertsVia<Sibling, AccountId>,
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
-	// Here/local root location to `AccountId`.
-	HashedDescription<AccountId, DescribeTerminus>,
```

Comment:
You cannot remove this as it breaks any existing accounts (e.g. any matching location that has already derived a local account using this converter).

You can add the new converter after this one, so any location that used to match and use this converter will continue to do so and not lose access to the derived account.

Reply:
reverted